cosim/black-parrot-example/dromajo_coemulation.hpp

```diff
@@ -0,0 +1,309 @@
+//
```

Comment:
I think this would be great to integrate into the zynq_pl_shell.hpp headers so it's platform specific.

cosim_init
cosim_step
cosim_trap
cosim_fini

FPGA will implement as fifo enqueue / dequeue through AXI, simulation can use backdoor access to the cosimulation module directly